tsdb/index/index.go

```diff
@@ -523,6 +523,11 @@ func (w *Writer) AddSymbol(sym string) error {
 }
 
 func (w *Writer) finishSymbols() error {
+	// Right now, the index symbol_table's <len> part is 4byte length. So the total symbol string length must less than 4byte
+ 	if w.f.pos - w.toc.Symbols - 4 > 4294967295 {
+		return errors.Wrap(errors.New("symbol length size exceeds 4byte:"),strconv.FormatInt(int64(w.f.pos-w.toc.Symbols-4),10))
```

Comment:
We could simplify this, errors.Wrap will render the error different and we usually use it to wrap other errors with a message.
```suggestion
		return errors.Errorf("symbol table size exceeds 4 bytes: %d", w.f.pos-w.toc.Symbols-4)
```